hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/utils/FrameBuilderTest.java

```diff
@@ -382,6 +387,7 @@ void constructsExpectedFrameForCreate() {
         assertEquals(transaction.gasAvailable(INTRINSIC_GAS), frame.getRemainingGas());
         assertSame(EIP_1014_ADDRESS, frame.getOriginatorAddress());
         assertEquals(Wei.of(NETWORK_GAS_PRICE), frame.getGasPrice());
+        assertEquals(Wei.ONE, frame.getBlobGasPrice());
```

Comment:
At first glance, it looks like all of these assertions are the same.  If they are can they be consolidated?

Reply:
I thought of refactoring that ... but (I admit it) just wanted to get it in.  Let me look agian.